### DISCUSSION TITLE
How to use SDDM lock screen?

### DISCUSSION THREAD

**Discussion by alimulap** [i3/i3#6018] [Q&A]
Honestly I barely know anything about these thing so I misunderstood a lot of things.

I usually use KDE Plasma + i3. But right now I'm trying to use only i3. 

The thing I'm very confused right now is how to use the login screen I usually see after locking the screen in KDE Plasma? I saw somewhere that I can use SDDM without DE like KDE Plasma. So I should be able to use it without KDE Plasma session right? 

The login screen when booting my laptop is obviously SDDM so why I can't see any tutorial to use that screen for lock screen?

Like if I use `i3lock` or `xscreensaver-command -lock` to lock the session, it has its own display for it. 

So, how to use SDDM display for the lock screen in i3wm?

Thank you!

**Comment by Chiarchiaooo** [comment]
As far as i know, there is a command that allows you to choose which login manager you want to use. 
It may vary for other distros, so try to google it first.

On ubuntu (and i think, also other debian-based distros) it's: `sudo dpkg-reconfigure lightdm`

**Reply by alimulap** [reply]
I am using arch. By following [this](https://wiki.archlinux.org/title/display_manager#Loading_the_display_manager), I already loading SDDM properly. And as I said, the display when I boot into the machine, it's using SDDM already, but not after I suspend my laptop. 

I am wondering how to use that "login when boot" screen for "login after suspend" screen.

**Comment by KebabLord** [comment]
unlike slim, LightDM, gdm etc. SDDM doesn't have a screenlock system as i know. This is not the fault of i3, SDDM just doesn't provide an x11 alternative to screenlocks.

Even if you manage to run SDDM login screen by a command with a workaround, i don't think it'd be safe since it could be crashed/bypassed etc. just like slimlock.

i strongly recommend you to use i3lock or xsecurelock by applying a custom theme instead.
